fabtests/common/shared.c

```diff
@@ -515,6 +521,9 @@ static int ft_alloc_msgs(void)
 		ret = ft_hmem_alloc(opts.iface, opts.device, (void **) &buf, buf_size);
 		if (ret)
 			return ret;
+
```

Comment:
Is it missing a `ft_hmem_alloc_host` here?

Reply:
yes, fixed in new revision.